### PR TITLE
Add sparql query for members of the House of Representatives of Nigeria

### PR DESCRIPTION
**Pull request by ondenman**
Part of: https://github.com/everypolitician/everypolitician-data/issues/27204

A member is no longer listed in any of the current categories being scraped as their membership has ceased. I have updated the member's wikidata item so that `position held` points to `Member of the House of Nigeria`. This PR adds a SPARQL query to fetch everyone with a P39 of Member of the House of Nigeria.

### CODE REVIEW

**Review by tmtmtmtm — CHANGES_REQUESTED** [review]
You've said _what_ you're doing in the PR description, and the commit message, but you haven't said _why_, which makes it very difficult to know whether this is actually the right thing to be doing.

**Comment by ondenman** [comment]
@tmtmtmtm I've updated the comment.

**Review by tmtmtmtm — CHANGES_REQUESTED** [review]
@ondenman I'm not sure the preconditions here hold true (i.e. that this member can't be in a relevant category just because their member has ceased). There's probably value in _also_ getting data by querying Wikidata anyway, but as we don't have historic terms yet, and you're only looking for a single member, it might be better to restrict this to only people who were members in the term we have.

File: scraper.rb [hunk 1]
```diff
@@ -2,12 +2,15 @@
 # encoding: utf-8
 
 require 'wikidata/fetcher'
+require 'pry'
```
Comment (tmtmtmtm) [inline scraper.rb]: Did you really mean to introduce this, or is this left over from local development/debugging? 

I don't really mind it being here, as it's always useful to have available, but if it's deliberate it should really be introduced in a separate commit.

File: scraper.rb [hunk 2]
```diff
@@ -2,12 +2,16 @@
...
 
 en = WikiData::Category.new('Category:Members of the House of Representatives (Nigeria)', 'en').member_titles |
      WikiData::Category.new('Category:Women members of the House of Representatives (Nigeria)', 'en').member_titles |
      WikiData::Category.new('Category:People\'s Democratic Party members of the House of Representatives (Nigeria)', 'en').member_titles |
      WikiData::Category.new('Category:Members of the House of Representatives (Nigeria) from Rivers State', 'en').member_titles |
-     WikiData::Category.new('Category:Speakers of the House of Representatives (Nigeria)', 'en').member_titles 
+     WikiData::Category.new('Category:Speakers of the House of Representatives (Nigeria)', 'en').member_titles
 
-EveryPolitician::Wikidata.scrape_wikidata(names: { en: en })
+sparq = 'SELECT ?item WHERE { ?item p:P39 ?position_held . ?position_held ps:P39 wd:Q21290864 . ?position_held pq:P2937 wd:Q20311259 }'
```
Comment (tmtmtmtm) [inline scraper.rb]: This is a little too complex to have on a single line, I fear. For legibility, and consistency with all the other scrapers where we do equivalent queries (and even just for it to be more usable on https://query.wikidata.org/, rather than most of it to be covered by the query explainer), it would be better to split this over multiple lines.

screen shot 2017-03-07 at 09 36 04

**Review by tmtmtmtm — CHANGES_REQUESTED** [review]

**Comment by ondenman** [comment]
@tmtmtmtm I've added two fixups. 3b8fa8c633b687baaeab3514b732a459e4efe2cc removes pry. 809017839baa6db80bc500187e5a89aa54b92377 splits the SPARQL query over multiple lines.

File: scraper.rb [hunk 3]
```diff
@@ -7,7 +7,16 @@
...
      WikiData::Category.new('Category:Members of the House of Representatives (Nigeria) from Rivers State', 'en').member_titles |
-     WikiData::Category.new('Category:Speakers of the House of Representatives (Nigeria)', 'en').member_titles 
+     WikiData::Category.new('Category:Speakers of the House of Representatives (Nigeria)', 'en').member_titles
 
-EveryPolitician::Wikidata.scrape_wikidata(names: { en: en })
+sparq = <<EOQ
+  SELECT ?item WHERE {
+    ?item p:P39 ?position_held .
+    ?position_held ps:P39 wd:Q21290864 .
+    ?position_held pq:P2937 wd:Q20311259
```
Comment (tmtmtmtm) [inline scraper.rb]: This is fine, but in case you were unaware, when you have two statements for the same subject, you can join them with a semicolon instead, i.e.:

```sparql
    ?position_held ps:P39 wd:Q21290864 ;
                   pq:P2937 wd:Q20311259 .
```

**Review by tmtmtmtm — COMMENTED** [review]
@ondenman an autosquash doesn't apply cleanly here — can you rebase this?

**Comment by ondenman** [comment]
@tmtmtmtm I've amended the query and rebased.

**Review by tmtmtmtm — COMMENTED** [review]
:+1:

(I've amended your commit to line the query up better)
